src/GlobalScreen/Scope/MetaBar/Factory/LinkItem.php

```diff
@@ -36,11 +38,12 @@ public function __construct(IdentificationInterface $provider_identification)
         $this->renderer = new LinkItemRenderer();
     }
     
-    public function withAction(string $action) : self
+    public function withAction(string $action, ?string $action_content_language = null) : self
```

Comment:
I'd like to have this as a separate method `withActionContentLanguage` since it's optional anyway. I see this is quite near to the actions itself but I do not like optional parameters and `Items` should be as simple as possible.